mm2src/coins/utxo/utxo_common.rs

```diff
@@ -4191,6 +4191,10 @@ pub fn derive_htlc_key_pair(coin: &UtxoCoinFields, _swap_unique_data: &[u8]) ->
     }
 }
 
+pub fn derive_htlc_pubkey(coin: &dyn SwapOps, swap_unique_data: &[u8]) -> Vec<u8> {
```

Comment:
Do you mean to un-inline `impl SwapOps for UtxoStandardCoin { #[inline] pub fn derive_htlc_pubkey() { utxo_common::derive_htlc_pubkey() } }`?
> un-inline the caller functions since this one is pub fn not like caller ones.

Reply:
> Do you mean to un-inline `impl SwapOps for UtxoStandardCoin { #[inline] pub fn derive_htlc_pubkey() { utxo_common::derive_htlc_pubkey() } }`?
> 
> > un-inline the caller functions since this one is pub fn not like caller ones.

Yes, I think keeping those as normal and making this one inlined simpler and better approach. And costs less on the compiler work.

Reply:
Then I don't understand when it's better to mark a trait method inline, and when better to not. At this comment https://github.com/KomodoPlatform/atomicDEX-API/pull/1496#discussion_r994353221 you suggested to inline all of the `SwapOps` trait methods. I'm confused 😁

Reply:
> Then I don't understand when it's better to mark a trait method inline, and when better to not.

It's nothing to do with trait method. My point is different. We have almost same function which is called from so many places. So inlining it will be more effective. 

> At this comment [#1496 (comment)](https://github.com/KomodoPlatform/atomicDEX-API/pull/1496#discussion_r994353221) you suggested to inline all of the `SwapOps` trait methods. I'm confused

I haven't seen this function were calling simple function which has very simple/short task/instruction.

Reply:
Done
Please note that I uninlined `SwapOps::derive_htlc_pubkey` and `SwapOps::derive_htlc_key_pair` implementations that call `utxo_common::derive_htlc_pubkey` and `utxo_common::derive_htlc_key_pair` correspondingly.